src/utils/boundingClientRect/index.js

```diff
@@ -14,6 +14,10 @@ const boundingClientRect: BoundingClientRect = ref => {
       right: right + (window.scrollX || window.pageXOffset),
       bottom: bottom + (window.scrollY || window.pageYOffset),
       left: left + (window.scrollX || window.pageXOffset),
+      pureTop: top,
```

Comment:
Later on, we should probably rename all returned values - maybe `relative` and `absolute` prefixes according to window.